helpers/cp_test.go

```diff
@@ -223,12 +224,10 @@ func (s *cpSuite) TestCopyPreserveAll(c *C) {
 
 	// Give the file a different mtime to ensure CopyFlagPreserveAll
 	// really works.
-	//
-	// You wonder why "touch" is used? And want to me about
-	// syscall.Utime()? Well, syscall not implemented on armhf
-	// Aha, syscall.Utimes() then? No, not implemented on arm64
-	// Really, this is a just a test, touch is good enough!
-	err = exec.Command("touch", src, "-d", "2007-08-23 08:21:42").Run()
+	err = unix.Utimes(src, []unix.Timeval{
+		unix.Timeval{Sec: 0, Usec: 0},
```

Comment:
[Also true in map literals from 1.5, fwiw](https://golang.org/doc/go1.5#map_literals).